rest/model/test.go

```diff
@@ -101,7 +101,7 @@ func (at *APITest) BuildFromService(st interface{}) error {
 		at.StartTime = utility.ToTimePtr(v.Start)
 		at.EndTime = utility.ToTimePtr(v.End)
 		duration := v.End.Sub(v.Start)
-		at.Duration = float64(duration)
+		at.Duration = float64(duration.Milliseconds())
```

Comment:
are you sure we're looking for milliseconds, though? In the previous case it looked to me like `duration` is the difference between two "python times" which are [in seconds](https://github.com/evergreen-ci/utility/blob/9e6c58062307caf4db647fba249bbe042b633ed8/time.go#L48).

Reply:
Oof nice catch.